nvflare/fuel/f3/communicator.py

```diff
@@ -177,17 +177,17 @@ def add_connector(self, url: str, mode: Mode, secure: bool = False) -> str:
             raise CommError(CommError.NOT_SUPPORTED, f"No driver found for URL {url}")
 
         params = parse_url(url)
-        return self.add_connector_advanced(driver_class(), mode, params, secure, False)
+        return self.add_connector_advanced(driver_class(), mode, params, secure, False), params
```

Comment:
The API may be simpler if we add a call to get params from a handle, like

`   params = connector.get_params(handle)`

Reply:
Maybe later.